client/app/scripts/superdesk/editor/editor.js

```diff
@@ -271,28 +389,51 @@ angular.module('superdesk.editor', [])
             templateUrl: 'scripts/superdesk/editor/views/editor.html',
             link: function(scope, elem, attrs, ngModel) {
 
-                var editorElem;
+                var editorElem, spelling = false, timeout;
```

Comment:
In a vacuum this is true, of course, but it might be different when the head is tired and you are switching contexts often.
(a bit of a guess, I admit, but it did happen to me when I was glancing over the code in a hurry and parsed the statement in the way described :) ... but in any case, it's an obsolete diff anyway)